src/auro-datetime.js

```diff
@@ -76,97 +64,84 @@ class AuroDatetime extends LitElement {
-  humanDateMonth() {
-    this.template.month = this.month;
-    const newDate = new Date();
+  humanDateConversion() {
+    let newDate = new Date();
 
-    return newDate.toLocaleString('en-us', this.template);
-  }
+    if (this.utc) {
+      this.template.timeZone = 'UTC';
+      newDate = new Date(this.utc);
+    } else if (this.setDate) {
+      newDate = new Date(this.setDate);
+    }
 
-  /**
-   * @private Internal function generate year date string
-   * @returns {string} - date string
-   */
-  humanDateYear() {
-    this.template.year = 'numeric';
-    const newDate = new Date();
+    switch (this.type) {
+      case 'day':
+        this.template.day = "numeric";
 
-    return newDate.toLocaleString('en-us', this.template);
-  }
+        return newDate.toLocaleString('en-us', this.template);
+      case 'month':
+        this.template.month = this.month;
 
-  /**
-   * @private Internal function generate weekday date string
-   * @returns {string} - date string
-   */
-  humanDateWeekday() {
-    this.template.weekday = this.weekday;
-    const newDate = new Date();
+        return newDate.toLocaleString('en-us', this.template);
+      case 'year':
+        this.template.year = 'numeric';
 
-    return newDate.toLocaleString('en-us', this.template);
-  }
+        return newDate.toLocaleString('en-us', this.template);
+      case 'weekday':
+        this.template.weekday = this.weekday;
 
-  /**
-   * @private Internal function generate day date string
-   * @returns {string} - date string
-   */
-  humanDateDay() {
-    this.template.day = "numeric";
-    const newDate = new Date();
+        return newDate.toLocaleString('en-us', this.template);
+
+      default: newDate.toLocaleString('en-us', this.template);
+    }
 
-    return newDate.toLocaleString('en-us', this.template);
+    return null
   }
 
   /**
-   * @private Internal function generate standard date string from input data
-   * @param {string} dateData - Input date data
+   * @private Internal function generate numeric date string 00/00/0000
    * @returns {string} - date string
    */
-  setHumanDate(dateData) {
-    const newDate = new Date(dateData);
+  numericDate() {
+    this.dateTemplate.month = 'numeric';
+    Reflect.deleteProperty(this.dateTemplate, 'weekday');
+    let newDate = new Date();
 
-    return newDate.toLocaleString('en-us', this.dateTemplate);
-  }
+    if (this.utc) {
+      this.dateTemplate.timeZone = 'UTC';
+      newDate = new Date(this.utc);
 
-  /**
-   * @private Internal function generate standard date string from UTC data
-   * @param {string} dateData - Input date data
-   * @returns {string} - date string
-   */
-  humanZuluDate(dateData) {
-    this.dateTemplate.timeZone = 'UTC';
-    const newDate = new Date(dateData);
+      return newDate.toLocaleString('en-us', this.dateTemplate);
+    } else if (this.setDate) {
+      newDate = new Date(this.setDate);
+
+      return newDate.toLocaleString('en-us', this.dateTemplate);
+    }
```

Comment:
We can use the one return statement at the end of the function.

```suggestion
    } else if (this.setDate) {
      newDate = new Date(this.setDate);
    }
```